conf/defaults.ini

```diff
@@ -582,6 +582,9 @@ oauth_auto_login = false
 # OAuth state max age cookie duration in seconds. Defaults to 600 seconds.
 oauth_state_cookie_max_age = 600
 
+# Sets a custom oAuth error message. This is useful if you need to point the users to a specific location for support.
+oauth_login_error_message =
```

Comment:
Even though we default to `oauth.login.error` when reading the config we should also set it here